packages/cli/src/utils/deploy-v2.ts

```diff
@@ -60,12 +64,20 @@ export async function deploy(mudConfig: MUDConfig, deployConfig: DeployConfig):
       World: worldContractName
         ? deployContractByName(worldContractName)
         : deployContract(WorldABI, WorldBytecode, "World"),
+      CoreModule: deployContract(CoreModuleData.abi, CoreModuleData.bytecode, "CoreModule"),
+      RegistrationModule: deployContract(RegistrationModuleData.abi, RegistrationModuleData.bytecode, "RootModule"),
```

Comment:
is this intentionally called `RootModule`?

Reply:
curious what this would have affected if left uncaught?

Reply:
It's just used for logging, the logs would have said "Deploying RootModule"